jira/client.py

```diff
@@ -4330,16 +4324,16 @@ def add_user(
             directoryId (int): The directory ID the new user should be a part of (Default: 1)
             password (Optional[str]): Optional, the password for the new user
             fullname (Optional[str]): Optional, the full name of the new user
-            notify (bool): Whether or not to send a notification to the new user. (Default: False)
-            active (bool): Whether or not to make the new user active upon creation. (Default: True)
-            ignore_existing (bool): Whether or not to ignore and existing user. (Default: False)
-            applicationKeys (Optional[list]): Keys of products user should have access to
+            notify (bool): Whether to send a notification to the new user. (Default: False)
```

Comment:
had to look it up (https://mcelroytutoring.com/blog-post.php?id=4165#:~:text=Here's%20the%20trick%3A%20%22Whether%20or,the%20outcome%20itself%20is%20undetermined.) but uneed seems better to remove the 'or not'